extensions/index.json

```
@@ -28,5 +28,5 @@
 		"videopress",
 		"wordads"
 	],
-	"beta": [ "amazon", "seo" ]
+	"beta": [ "amazon", "seo", "cool-block" ]
```

You probably want to remove that test block.

```suggestion
	"beta": [ "amazon", "seo" ]
```